.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: 🐛 Bug Report
+description: Create a report to help us improve
+labels: ["bug", "needs-triage"]
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      **Thank you for wanting to report a bug in pyhf!**
+
+
+      ⚠
+      Verify first that your issue is not [already reported on
+      GitHub][issue search].
+
+
+      [issue search]: https://github.com/scikit-hep/pyhf/issues?q=is%3Aopen+is%3Aissue+label%3Abug
+
+- type: textarea
+  attributes:
+    label: Summary
+    description: Explain the problem briefly below.
+    placeholder: >-
+      When I try to do X with pyhf and the following workspace, Y breaks or
+      Z happens in an unexpected manner.
+      Here are all the details I know about this problem.
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: pyhf Version
+    description: >-
+      Paste verbatim output from `pyhf --version` below, under
+      the prompt line. Please don't wrap it with tripple backticks &mdash; your
+      whole input will be turned into a code snippet automatically.
+    render: console
+    placeholder: |
+      $ pyhf --version
+      pyhf, version 0.6.2
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: OS / Environment
+    description: >-
+      Provide the name of your operating system and the output of `uname --all`.
```

Comment:
This part seems like it would not matter for the large majority of existing issues. I think it does not hurt to have for completeness, but would suggest moving to the end of the fields. `uname --all` will not always work, for example on Mac OS I need `uname -a`.